packages/modular-scripts/react-scripts/config/utils/esmUtils.js

```diff
@@ -8,7 +8,7 @@ function parsePackageName(name) {
   if (!parsedName) {
     return;
   }
-  const [_, scope, module, submodule] = parsedName;
```

Comment:
Think it was `eslint` that was upset about the existence of this underscore, for some reason.